content/en/docs/started/workstation/minikf-gcp.md

```diff
@@ -3,12 +3,6 @@ title = "Deploy using MiniKF on GCP"
 description = "Using Google Cloud Marketplace to deploy MiniKF (mini Kubeflow) on Google Cloud Platform (GCP)"
```

Comment:
Thanks @cspavlou !

A few suggestions, since we're in this guide:

- Use "Google Cloud" (rebranding?) instead of "...Platform"
- Change the order of the words in the description

```suggestion
description = "Deploy Kubeflow with MiniKF (mini Kubeflow) via Google Cloud Marketplace"
```

cc @alfsuse @RFMVasconcelos 

Also, in the title (couldn't highlight directly):

- Let's be a bit more descriptive

```diff
- title = "Deploy using MiniKF on GCP"		
+ title = "Deploy Kubeflow using MiniKF on Google Cloud"
```

LMKWYT

Reply:
Line 16 and elsewhere: "Google Cloud" <- "GCP", if possible

Reply:
Thanks for your comments! I will update accordingly